pkg/generate/generate.go

```diff
@@ -53,14 +53,14 @@ func (c *Controller) applyGenerate(resource unstructured.Unstructured, gr kyvern
 	if err != nil {
 		if apierrors.IsNotFound(err) {
 			for _, e := range gr.Status.GeneratedResources {
-				resp, err := c.client.GetResource(e.Kind, e.Namespace, e.Name)
+				resp, err := c.client.GetResource(e.APIVersion, e.Kind, e.Namespace, e.Name)
 				if err != nil {
 					logger.Error(err, "Generated resource failed to get", "Resource", e.Name)
 				}
 
 				labels := resp.GetLabels()
-				if labels["policy.kyverno.io/synchronize"] == "enable" {
-					if err := c.client.DeleteResource(resp.GetKind(), resp.GetNamespace(), resp.GetName(), false); err != nil {
+				if labels["app.kubernetes.io/synchronize"] == "enable" {
```

Comment:
This label is used in other places, should we change all of them? If so,  let's make it a constant value.

Reply:
It is used separately only in this file.

Reply:
Is this used to identify Kyverno `synchronize` flag? I don't see we set it anywhere.

Reply:
yes correct, It's been modified. I reverted it back.